modules/high-sev-security-policies.adoc

```diff
@@ -18,12 +18,13 @@ The following table lists the default security policies in {product-title} that
 [options="header"]
 |====
 |Life cycle stage |Name |Description | Status
-|Build or Deploy |Fixable CVSS >= 6 and Privileged |Alerts when deployments run in privileged mode with fixable vulnerabilities that have a CVSS of at least 6. |Enabled
-|Build or Deploy |Fixable CVSS >= 7 |Alerts when deployments with fixable vulnerabilities have a CVSS of at least 7. |Enabled
+|Build or Deploy |Fixable CVSS >= 7 |Alerts when deployments with fixable vulnerabilities have a CVSS of at least 7. |Disabled
 |Build or Deploy |Fixable Severity at least Important |Alerts when deployments with fixable vulnerabilities have a severity rating of at least Important. |Enabled
 |Build or Deploy |Secure Shell (ssh) Port Exposed in Image |Alerts when deployments expose port 22, which is commonly reserved for SSH access. |Enabled
 |Deploy |Emergency Deployment Annotation |Alerts when deployments use the emergency annotation, such as "admission.stackrox.io/break-glass":"ticket-1234" to circumvent StackRox Admission Controller checks. |Enabled
 |Deploy |Environment Variable Contains Secret |Alerts when deployments have environment variables that contain 'SECRET'. |Enabled
+|Deploy |Fixable CVSS >= 6 and Privileged |Alerts when deployments run in privileged mode with fixable vulnerabilities that have a CVSS of at least 6. |Disabled
+|Deploy | Privileged Containers with Important and Critical Fixable CVEs |Alerts when containers have important or critical fixable vulnerabilities and are running in privileged mode. Known vulnerabilities make it easier for adversaries to exploit your application, and highly-privileged containers pose greater risk. You can fix these high-severity vulnerabilities by updating to a newer version of the affected components or running your container with lower privileges. |Enabled
```

Comment:
For other policies it just lists the description. This one has description, rationale and remediation all mixed in. Is there a reason to not be consistent?

Reply:
Not a good reason. Mostly a copy/paste error. I'll update this one to be consistent with the others. Thanks!